curriculum/challenges/english/10-coding-interview-prep/data-structures/create-a-stack-class.md

```diff
@@ -89,8 +90,9 @@ The `pop` method should remove and return the top element of the stack
 assert(
   (function () {
     var test = new Stack();
+    test.push('CS61');
     test.push('CS50');
-    return test.pop() === 'CS50';
+    return test.pop() === 'CS50' && test.pop() == 'CS61';
```

Comment:
Just a little nitpick - let's make second comparison also strict (`===`) for consistency.

Reply:
Whoops - that was a silly typo! Added another commit to resolve that.